docs/cli/operator.mdx

```diff
@@ -573,30 +573,30 @@ Use the options listed below to change the command's behavior.
 
 ## nexus
 
-Nexus commands enable managing Nexus resources.
+These commands manage Nexus resources.
 
-Nexus commands follow this syntax:
+nexus commands follow this syntax:
 
 ```
-temporal operator nexus [command] [command] [command options]
+temporal operator nexus [command] [subcommand] [options]
 ```
 
 ### endpoint
 
-Endpoint commands enable managing Nexus Endpoints.
+These commands manage Nexus Endpoints.
 
-Endpoint commands follow this syntax:
+Nexus Endpoint commands follow this syntax:
 
 ```
-temporal operator nexus endpoint [command] [command options]
+temporal operator nexus endpoint [command] [options]
 ```
 
 #### create
 
-Create a new Nexus Endpoint on the Server.
+Create a Nexus Endpoint on the Server.
 
-An endpoint name is used in Workflow code to invoke Nexus operations.
-The endpoint target may either be a Worker, in which case `--target-namespace` and
+A Nexus Endpoint name is used in Workflow code to invoke Nexus Operations.  The
```

Comment:
And put every sentence on one line. 
Running `yarn format` will also help with formatting. We run it before merging.